### ISSUE TITLE
Change chat widget script

### ISSUE THREAD

**Issue by cuevasm** [blockstack/app.co#494]
Hey team, I need this swapped out please, replace the Crisp code with: 

'''
<script>
window.HFCHAT_CONFIG = {
  EMBED_TOKEN: '4b938070-600d-11e9-a17f-8d51a82b8e0a',
  ASSETS_URL: 'https://widget.happyfoxchat.com/visitor'
};
(function () {
  var scriptTag = document.createElement('script')
  scriptTag.type = 'text/javascript'
  scriptTag.async = true
  scriptTag.src = window.HFCHAT_CONFIG.ASSETS_URL + '/js/widget-loader.js'

  var s = document.getElementsByTagName('script')[0]
  s.parentNode.insertBefore(scriptTag, s)
})()
</script>

'''

There was [good reason for this change](https://paper.dropbox.com/doc/Chat-Support-at-Blockstack--AbfF9NV~x7x__nScr~CAOcQ1Ag-JeGdgbgO7ayDZhgGN27kv), read here if you want to know more.

**Comment by hstove** [comment]
Just updating here: I've switched out the code in a branch, but we need to have it so this chat widget only displays on /mining.

**Comment by cuevasm** [comment]
Yeah I mentioned I can't control that from this side with this tool. You can hold off, in having trouble with this service right now anyway. I'll circle back at the end of the week

**Comment by markmhx** [comment]
@cuevasm Do you want to proceed with this issue or should we close it out?
